docs/network-mobile/mobile-proof-of-coverage.mdx

```diff
@@ -25,7 +25,7 @@ that Hotspots are honestly representing their location and the wireless network
 creating from that location.
 
 While current 5G Hotspots are _also_ able to participate in the IOT Network and PoC, the CBRS Small
-Cell and 5G aspects of a 5G Hotspot require a new approach to validating PoC.
+Cell and Wi-Fi Access Points require a new approach to validating PoC.
```

Comment:
Should clarify that current CBRS Hotspots can participate in IoT, but WiFi cannot.